### ISSUE TITLE
Use PDS tools to read PDS4 labels

### ISSUE THREAD

**Issue by jessemapel** [USGS-Astrogeology/ale#191]
We should consider using the PDS4 tools (https://github.com/Small-Bodies-Node/pds4_tools) to read pds4 labels. I'm not sure how much it will help with but it makes sense to use the tools from the people who make and update the standard.

**Comment by thareUSGS** [comment]
+1. It would be good to support that effort and give back any updates we might provide. Like planetaryimage the only issue we might have is handling huge data sets (which is going to fairly rare for the ALE use case).

**Comment by jessemapel** [comment]
@thareUSGS ALE doesn't hit the image data all, we're purely pulling the metadata off the label so that shouldn't be a problem.

**Comment by thareUSGS** [comment]
Right - good point. I just notice the "version" of the tool you pointed at. This original Small Body code base has been taken over by the PDS Engineering Node and I think also incorporated PDS3 label support. I would recommend that we provide feedback to this version of the code. https://github.com/NASA-PDS-Incubator/pds-view

**Comment by jlaura** [comment]
:+1: Docs look good too which makes usage much more straight forward. Is this the package that they are cool with us pushing to Conda-forge? If so, that would be awesome and makes this a win/win for us looking to parse and support pds4 labels inside of ALE.

**Comment by jessemapel** [comment]
@jlaura Yes, they said just make sure we post an issue so everyone knows what's going on. Also @thareUSGS I've been told that both repos are basically the same.

**Comment by jlaura** [comment]
:+1: Let's do it in the next sprint then if that makes sense. The total lack of dependencies makes this a piece of cake too. We can totally build it on Conda-forge with matplotlib and tkinter too.

We should use the first linked repo because they have a 1.1 release tagged. The other repo doesn't
